README.md

```diff
@@ -123,6 +123,9 @@ require 'window-picker'.setup({
         -- selection or not
         include_current_win = false,
 
+		-- whether to include windows marked as unfocusable
+		include_unfocusable_windows = false,
+
```

Comment:
Rest of the README seems to be indented with spaces instead of tabs. Don't want to take a dive into that particular can of worms but I think it would be better to stay consistent with the existing formatting.

Reply:
Oops. :) I copy-pasted from the lua file without paying attention.

Reply:
Been there 😅